shell/common/shell_test_external_view_embedder.cc

```diff
@@ -44,7 +44,9 @@ void ShellTestExternalViewEmbedder::BeginFrame(
     SkISize frame_size,
     GrDirectContext* context,
     double device_pixel_ratio,
-    fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger) {}
+    fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger) {
+  visited_platform_views_.clear();
```

Comment:
Should `current_composition_params_` and `mutators_stacks_` also be cleared?

Reply:
True, fixed.